hostinit/setup.go

```diff
@@ -367,13 +366,20 @@ func (init *HostInit) IsHostReady(host *host.Host) (bool, error) {
 	if err != nil {
 		return false, errors.Wrapf(err, "failed to get cloud host for %s", host.Id)
 	}
-	reachable, err := cloudHost.IsSSHReachable()
-	if err != nil {
-		return false, errors.Wrapf(err, "error checking if host %s is reachable", host.Id)
+	reachableTimes := 0
+	for {
```

Comment:
Nit, but i'd use a normal for loop here,
`for reachableTimes := 1; reachableTimes < 3; i++ ` and but the non-error returns outside of the loop